<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Nosto</class>
 <widget class="QDialog" name="Nosto">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>572</width>
    <height>1314</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="lblkayttaja">
     <property name="font">
      <font>
       <pointsize>48</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn20">
     <property name="font">
      <font>
       <pointsize>48</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {

  background-color:  #C0C0C0;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    min-width: 10em;
    padding: 6px;
	color: black;
}♣

QPushButton:pressed {

  background-color:  #808080;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    min-width: 10em;
    padding: 6px;
	color: black;
}♣QPushButton {

  background-color:  #C0C0C0;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    min-width: 10em;
    padding: 6px;
	color: black;
}♣

QPushButton:pressed {

  background-color:  #808080;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    min-width: 10em;
    padding: 6px;
	color: black;
}♣</string>
     </property>
     <property name="text">
      <string>20</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
     <property name="autoRepeatInterval">
      <number>100</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn40">
     <property name="font">
      <font>
       <pointsize>48</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {

  background-color:  #C0C0C0;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    min-width: 10em;
    padding: 6px;
	color: black;
}♣

QPushButton:pressed {

  background-color:  #808080;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    min-width: 10em;
    padding: 6px;
	color: black;
}♣QPushButton {

  background-color:  #C0C0C0;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    min-width: 10em;
    padding: 6px;
	color: black;
}♣

QPushButton:pressed {

  background-color:  #808080;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    min-width: 10em;
    padding: 6px;
	color: black;
}♣</string>
     </property>
     <property name="text">
      <string>40</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn60">
     <property name="font">
      <font>
       <pointsize>48</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {

  background-color:  #C0C0C0;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    min-width: 10em;
    padding: 6px;
	color: black;
}♣

QPushButton:pressed {

  background-color:  #808080;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    min-width: 10em;
    padding: 6px;
	color: black;
}♣QPushButton {

  background-color:  #C0C0C0;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    min-width: 10em;
    padding: 6px;
	color: black;
}♣

QPushButton:pressed {

  background-color:  #808080;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    min-width: 10em;
    padding: 6px;
	color: black;
}♣</string>
     </property>
     <property name="text">
      <string>60</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn100">
     <property name="font">
      <font>
       <pointsize>48</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {

  background-color:  #C0C0C0;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    min-width: 10em;
    padding: 6px;
	color: black;
}♣

QPushButton:pressed {

  background-color:  #808080;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    min-width: 10em;
    padding: 6px;
	color: black;
}♣QPushButton {

  background-color:  #C0C0C0;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    min-width: 10em;
    padding: 6px;
	color: black;
}♣

QPushButton:pressed {

  background-color:  #808080;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    min-width: 10em;
    padding: 6px;
	color: black;
}♣</string>
     </property>
     <property name="text">
      <string>100</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn200">
     <property name="font">
      <font>
       <pointsize>48</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {

  background-color:  #C0C0C0;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    min-width: 10em;
    padding: 6px;
	color: black;
}♣

QPushButton:pressed {

  background-color:  #808080;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    min-width: 10em;
    padding: 6px;
	color: black;
}♣QPushButton {

  background-color:  #C0C0C0;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    min-width: 10em;
    padding: 6px;
	color: black;
}♣

QPushButton:pressed {

  background-color:  #808080;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    min-width: 10em;
    padding: 6px;
	color: black;
}♣</string>
     </property>
     <property name="text">
      <string>200</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn500">
     <property name="font">
      <font>
       <pointsize>48</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {

  background-color:  #C0C0C0;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    min-width: 10em;
    padding: 6px;
	color: black;
}♣

QPushButton:pressed {

  background-color:  #808080;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    min-width: 10em;
    padding: 6px;
	color: black;
}♣QPushButton {

  background-color:  #C0C0C0;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    min-width: 10em;
    padding: 6px;
	color: black;
}♣

QPushButton:pressed {

  background-color:  #808080;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    min-width: 10em;
    padding: 6px;
	color: black;
}♣</string>
     </property>
     <property name="text">
      <string>500</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblDebit_nostosumma">
     <property name="font">
      <font>
       <pointsize>48</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Nostettava summa:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="Debitdisplay">
     <property name="font">
      <font>
       <pointsize>48</pointsize>
      </font>
     </property>
     <property name="maxLength">
      <number>3</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>48</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Muu summa:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="muuSumma">
     <property name="font">
      <font>
       <pointsize>48</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnNosta">
     <property name="font">
      <font>
       <pointsize>48</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {

  background-color:  #C0C0C0;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    min-width: 10em;
    padding: 6px;
	color: black;
}♣

QPushButton:pressed {

  background-color:  #808080;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    min-width: 10em;
    padding: 6px;
	color: black;
}♣QPushButton {

  background-color:  #C0C0C0;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    min-width: 10em;
    padding: 6px;
	color: black;
}♣

QPushButton:pressed {

  background-color:  #808080;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    min-width: 10em;
    padding: 6px;
	color: black;
}♣</string>
     </property>
     <property name="text">
      <string>Nosta</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnMenu">
     <property name="font">
      <font>
       <pointsize>48</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {

  background-color:  #C0C0C0;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    min-width: 10em;
    padding: 6px;
	color: black;
}♣

QPushButton:pressed {

  background-color:  #808080;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    min-width: 10em;
    padding: 6px;
	color: black;
}♣QPushButton {

  background-color:  #C0C0C0;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    min-width: 10em;
    padding: 6px;
	color: black;
}♣

QPushButton:pressed {

  background-color:  #808080;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    min-width: 10em;
    padding: 6px;
	color: black;
}♣</string>
     </property>
     <property name="text">
      <string>Menu</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblInfo">
     <property name="font">
      <font>
       <pointsize>28</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
